tony-core/src/main/java/com/linkedin/tony/TonyApplicationMaster.java

```diff
@@ -295,6 +332,8 @@ private void buildTensorFlowSession() {
    */
   public static void main(String[] args) {
     boolean succeeded = false;
+    long started;
+    long completed;
```

Comment:
I addition to start time (which is when this code runs), it would be informative to know the _submit_ time as well, which is when YARN received the AM container request. I think you should be able to get that from `amRMClient.getStartTime()`

Reply:
I'm assuming that info (submit time) would be on the job listing page as well?

Reply:
(pasting reply here from below)

Yes, I think it would be informative to show the submit time on the job listing page as well. It let's us calculate the delay between submit time and start time.

Reply:
Ok I'll put it on the back burner for the future PRs.